projects/step-frontend/src/lib/modules/timeseries/pages/dashboard/chart-dashlet/settings/chart-dashlet-settings.component.ts

```diff
@@ -0,0 +1,100 @@
+import { Component, inject, OnInit } from '@angular/core';
```

Comment:
1. If form's creation methods exists inside the component, there is no sense to put the form builder as fb parameter.
But it might be useful to place form types and form construction functions in the separate file (Search for assert-metric.form as example)

2. Instead of duplicating form structure both in types and functions, you can automatically extract the type from function.
```
export type ChartForm = ReturnType<typeof createFormGroup>;

export const  createFormGroup = (fb: FormBuilder, item: DashboardItem) => {
    return fb.group({
      name: fb.control(item.name, [Validators.required]),
      chartSettings: item.chartSettings ? createChartSettingsGroup(fb, item.chartSettings) : null,
      size: fb.control(item.size, [Validators.required, Validators.min(1)]),
    });
 };
```